vnext/Desktop.IntegrationTests/React.Windows.Desktop.IntegrationTests.vcxproj

```diff
@@ -64,35 +64,30 @@
       <PreprocessorDefinitions>OSS_RN;FOLLY_MOBILE=1;ENUM_BITFIELDS_NOT_SUPPORTED;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
   </ItemDefinitionGroup>
-  <ItemDefinitionGroup Condition="'$(Configuration)'=='Debug'">
-    <ClCompile>
-      <PreprocessorDefinitions>_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-    </ClCompile>
-  </ItemDefinitionGroup>
-  <ItemDefinitionGroup Condition="'$(Configuration)'=='Release'">
-    <ClCompile>
-      <Optimization>MaxSpeed</Optimization>
```

Comment:
><Optimization>MaxSpeed</Optimization> [](start = 6, length = 37)

Is this intentional to remove these .. This will result in unoptimized release binaries, is it ?

Reply:
+1. Are these settings perhaps contained in a shared prop sheet?

---
In reply to: [297263383](https://github.com/microsoft/react-native-windows/pull/2670#discussion_r297263383) [](ancestors = 297263383)

Reply:
Indeed. See https://github.com/microsoft/react-native-windows/blob/master/vnext/PropertySheets/Release.props